magento/magento2
# [MFTF] AdminMediaGalleryCatalogUiEditCategoryGridPageTest failing randomly

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1764: [MFTF] AdminMediaGalleryCatalogUiEditCategoryGridPageTest failing randomly 

### Manual testing scenarios (*)
Run `AdminMediaGalleryCatalogUiEditCategoryGridPageTest`

Correct category edit page should be opened

### Questions or comments


### Contribution checklist (*)
